modules/zio/src/main/scala/scalacache/zio.scala

```diff
@@ -0,0 +1,48 @@
+package scalacache
+
+import _root_.zio.Task
+
+import scala.util.control.NonFatal
+
+object zio {
```

Comment:
Can we have this as `ZioEffect` similar to `scalacache.CatsEffect`.
It is common to `import scalacache._` and this ends up shading the `zio` namespace.

Reply:
Thanks for your feedback. That sounds reasonable. I renamed the module object according your suggestion.